wp-graphql.php

```diff
@@ -398,6 +398,13 @@ public static function show_in_graphql() {
 				$wp_taxonomies['post_tag']->graphql_single_name = 'tag';
 				$wp_taxonomies['post_tag']->graphql_plural_name = 'tags';
 			}
+
+			// Adds GraphQL support for post formats.
+			if ( isset( $wp_taxonomies['post_format'] ) ) {
+				$wp_taxonomies['post_format']->show_in_graphql = true;
+				$wp_taxonomies['post_format']->graphql_single_name = 'PostFormat';
```

Comment:
Probably should start with lower case `p` to match the convention on other taxonomies. Ex. `mediaItem` & `mediaItems`

Reply:
you're absolutely right. I'll fix that now